index.html

```diff
@@ -4,6 +4,10 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+<h1>
+  Classical Music
```

Comment:
Watch out for extraneous white space.
The `<img>` tag is missing an equals sign.
The `<h1>` tag should only encapsulate your `Classical Music` phrase.

Change to the following:
```html
  <h1>Classical Music</h1>
  <img src="https://media.giphy.com/media/11Lz1Y4n1f2j96/giphy.gif">
```